File: 121 felipez java/punto/src/Punto.java
public class Punto {
    private float x;
    private float y;

    public Punto(float x, float y) {
        this.x = x;
        this.y = y;
    }

    public String coordCartesianas() {
        return "(" + x + ", " + y + ")";
    }

    public String coordPolares() {
        double r = Math.sqrt(x * x + y * y);
        double theta = Math.toDegrees(Math.atan2(y, x));
        return "(" + r + ", " + theta + "°)";
    }

    public String toString() {
        return "Punto(" + x + ", " + y + ")";
    }

    public static void main(String[] args) {
        Punto p = new Punto(3, 4);
        System.out.println("Coordenadas cartesianas: " + p.coordCartesianas());
        System.out.println("Coordenadas polares: " + p.coordPolares());
        System.out.println("Representación: " + p);
    }
}

File: Practica 1/punto.py
import math

class Punto:
    def __init__(self, x: float, y: float):
        self.x = x
        self.y = y

    def coord_cartesianas(self):
        return self.x, self.y

    def coord_polares(self):
        r = math.sqrt(self.x**2 + self.y**2)
        theta = math.atan2(self.y, self.x)
        return r, math.degrees(theta)

    def __str__(self):
        return f"Punto({self.x}, {self.y})"

p = Punto(3, 4)
print("Coordenadas cartesianas:", p.coord_cartesianas())
print("Coordenadas polares:", p.coord_polares())
print("Representación:", p)
